firebase-installations-interop/src/main/java/com/google/firebase/installations/InstallationTokenResult.java

```diff
@@ -46,7 +45,7 @@
     public abstract Builder setToken(@NonNull String value);
 
     @NonNull
-    public abstract Builder setTokenExpirationTimestampMillis(@NonNull long value);
+    public abstract Builder setTokenExpirationInSecs(@NonNull long value);
```

Comment:
I guess this is the source of the generated API txt doc

Reply:
Done. Thanks.